### PR TITLE
Bump the ansible-lint version to 24.7.0 and ansible version to min 2.15 

### PR DESCRIPTION
**Pull request by mandar242**
Bump the ansible-lint version to 24.7.0 and ansible version to min 2.15

### CODE REVIEW

File: .github/workflows/linters.yaml [hunk 1]
```diff
@@ -17,8 +17,8 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Run ansible-lint
-        uses: ansible/ansible-lint@v6.21.0
+      - name: run-ansible-lint
```
Comment (alinabuzachis) [inline .github/workflows/linters.yaml:20]: Why the change of step name?

Comment (mandar242) [reply .github/workflows/linters.yaml:20]: Replaced it with new task from https://github.com/marketplace/actions/run-ansible-lint
image

**Review by alinabuzachis — APPROVED** [review]

**Review by GomathiselviS — APPROVED** [review]
